qemu/tests/blockdev_commit_powerdown.py

```diff
@@ -0,0 +1,66 @@
+from virttest import env_process
+
+from provider import job_utils
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitHotunplug(BlockDevCommitTest):
+
+    def commit_snapshot_and_destory_vm(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["base-node", "top-node", "speed"]
+        arguments = self.params.copy_from_keys(options)
+        device = self.get_node_name(snapshot_tags[-1])
+        commit_cmd = backup_utils.block_commit_qmp_cmd
+        cmd, args = commit_cmd(device, **arguments)
+        self.main_vm.monitor.cmd(cmd, args)
+        self.main_vm.destroy(gracefully=False)
+
+    def boot_and_commit_snapshot(self):
+        vm_name = self.params["main_vm"]
+        vm_params = self.params.object_params(vm_name)
+        images = self.params.objects("images")
+        vm_params["images"] = " ".join(images)
+        env_process.preprocess_vm(self.test, vm_params, self.env, vm_name)
+        self.main_vm = self.env.get_vm(vm_name)
+        self.pre_test()
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["base-node", "top-node", "speed"]
+        arguments = self.params.copy_from_keys(options)
+        device = self.get_node_name(snapshot_tags[-1])
+        commit_cmd = backup_utils.block_commit_qmp_cmd
+        cmd, args = commit_cmd(device, **arguments)
+        self.main_vm.monitor.cmd(cmd, args)
+        job_id = args.get("job-id", device)
+        job_utils.wait_until_block_job_completed(self.main_vm, job_id)
```

Comment:
can we call backup_utils.block_commit here?

Reply:
Ok,Thanks.